### PR TITLE
Get api articles idarticle

### PR DESCRIPTION
**Pull request by Jlh77**
Still waiting for 2 pull requests prior

### CODE REVIEW

File: __tests__/api.test.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const testData = require("../db/data/test-data");
+const request = require("supertest");
+const app = require("../app.js");
+
+afterAll(() => db.end());
+
```
Comment (karlaferry) [inline __tests__/api.test.js:10]: Don't forget beforeEach :)

Comment (Jlh77) [reply __tests__/api.test.js:10]: ah yes just added it

File: models/api.model.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+const db = require("../db/connection");
+
+exports.selectAllTopics = () => {
+  return db.query("SELECT * FROM topics").then((res) => {
+    return res.rows;
+  });
+};
+
+exports.selectArticleById = async (article_id) => {
```
Comment (karlaferry) [inline models/api.model.js:8]: Good practice of async/await, however I would suggest that you stay consistent; if you're using async/await in your models, it's best practice to use it with all of it!

**Review by karlaferry — CHANGES_REQUESTED** [review]
Try to be consistent with the usage of async/await. Once you've changed it feel free to merge :)

**Pull request merged**